### ISSUE TITLE
Names of some desktop entries are showing as untranslated

### ISSUE THREAD

**Issue by AsciiWolf** [linuxmint/mintmenu#250]
Some desktop entries have untranslated (English) names on my non-English (Czech), fully updated, Linux Mint 19.3 MATE system. All applications are fully translated and do not have this issue on Ubuntu 18.04. One of these applications is "Disk Usage Analyzer".

**Comment by clefebvre** [comment]
Make sure language-pack-gnome-cs language-pack-cs are installed.

mintreport and mintlocale should report these as missing..

**Comment by AsciiWolf** [comment]
Hello Clement,

I have already deleted the testing vm, however I am quite sure that the language packs were installed. The system was installed using a latest Linux Mint 19.3 MATE 64-bit iso with "Czech" language selected during installation, then fully updated using the GUI Update Manager. All applications that I tried were correctly translated and I did not notice anything localization-related mentioned in mintreport. The only thing that was in English although it is translated in upstream/Ubuntu was desktop entries for "Disk Usage Analyzer" and a few other applications in the Mint menu.

**Comment by AsciiWolf** [comment]
So I have tried installing a new vm and can confirm that there is still the same issue:

![mintmenu_translation_issue](https://user-images.githubusercontent.com/2339930/80320037-30f04e80-8814-11ea-91e1-118a549436e6.png)

Language packs are correctly installed:
```
ii  language-pack-cs                           1:18.04+20180712
ii  language-pack-cs-base                      1:18.04+20180712
ii  language-pack-gnome-cs                     1:18.04+20180712
ii  language-pack-gnome-cs-base                1:18.04+20180712
```

**Comment by clefebvre** [comment]
I can see the translation in language-package-gnome-cs-base (in /usr/share/locale-langpack/).

Can you reproduce this with another menu (there are some available as MATE applets) and/or with other locales?

**Comment by clefebvre** [comment]
oh wait... I got confused between gnome-disks and baobab..

**Comment by clefebvre** [comment]
ok so for Dictionary and Baobab, the translations are directly in the desktop file.. you don't even need the language pack.

**Comment by clefebvre** [comment]
```
Name[cs]=Slovník
Comment[cs]=Kontrolovat slova a pravopis v on-line slovníku
```

```
Name[cs]=Analyzátor využití disku
Comment[cs]=Zkontrolovat velikost složek a dostupné místo na disku
```

**Comment by clefebvre** [comment]
Leave it with me, I'll need to test this in MATE with the right locale.

**Comment by AsciiWolf** [comment]
Feel free to let me know if you need any additional testing or other help. :-)

**Comment by clefebvre** [comment]
Hi @AsciiWolf, sorry it took me so long, I had to prioritize other issues before coming back to this one. It should have taken me no time at all, but I didn't realize these were MATE forks... I was looking at the GNOME dictionary and baobab, and these are well translated in Czech.

So anyway, I found the issue. It isn't in mintmenu. Czech translations are missing in the desktop files for MATE dictionary and the MATE disk usage analyzer. These should be reported to MATE directly.
